src/api_client.rs

```diff
@@ -474,10 +474,22 @@ impl Client {
 
     /* validator namespace */
     pub async fn get_attester_duties(
+        &self,
         epoch: Epoch,
         indices: &[ValidatorIndex],
     ) -> Result<(Root, Vec<AttestationDuty>), Error> {
-        unimplemented!("")
+        let endpoint = format!("eth/v1/validator/duties/attester/{epoch}");
+        let mut result: Value<Vec<AttestationDuty>> = self
+            .http_post(&endpoint, indices)
+            .await?
+            .json()
```

Comment:
here we should check for an API error...

and I would also ideally like to re-use the `post`/`http_post` code

I *think* we should make `post` more general so that it returns some other return type `U`

this implies changing `api_error_or_ok` which affects the public interface of this crate -- and this is not the end of the world but it is a bit of an involved change...

let me double-check if this is worth it, based on how many functions use HTTP POST to return a value or if this one method is in the minority...

Reply:
only the attester duties and sync committee duties follow this pattern so I think we will just make exceptions for them here!